bin/debug/request.ts

```diff
@@ -10,8 +10,8 @@ export class DebugRequest {
 
     constructor(public path = '') {
         this.program
-            .option('-m, --member <member>', 'member slug')
-            .option('-o, --origin <origin>', 'base URL to send the request to')
```

Comment:
Small nit: This change to origin need to be reflected where the option is read below.

Reply:
ah, yep good point. ty!  I'll address this in a follow up.